packages/mobile/lib/auxiliary/drawer/actions/openDrawer.action.ts

```diff
@@ -0,0 +1,8 @@
+import { DrawerId } from '../enums'
+import { drawersStore } from '../stores'
+import { DrawerProps } from '../types'
+
+export function openDrawer(drawerId: DrawerId, props?: DrawerProps): void {
```

Comment:
Same as the comment for `closeDrawer` - we can extract the logic here to a `addDrawerState` function that is called in `openDrawer`. Would be good in case we ever need to do extra logic besides actually updating the store itself.